keyboards/doio/kb12/info.json

```diff
@@ -0,0 +1,38 @@
+{
+    "keyboard_name": "KB12-01",
+    "manufacturer": "DOIO",
+    "url": "",
+    "maintainer": "Alice",
```

Comment:
This should probably be you GitHub name

Reply:
@elpekenin  Do you mean the "DOIO" change "DOIO2022"?